<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sysmessage</class>
 <widget class="QDialog" name="sysmessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="screen_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>431</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="code_title_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Code :</string>
   </property>
  </widget>
  <widget class="QLabel" name="code_data_label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>000</string>
   </property>
  </widget>
  <widget class="QLabel" name="Message_title_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Message :</string>
   </property>
  </widget>
  <widget class="QLabel" name="Message_data_label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>000</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
